sourcegraph/sourcegraph
# Fix ref panel action button

## Description
Addresses https://github.com/sourcegraph/sourcegraph/issues/39517.


https://user-images.githubusercontent.com/2196347/185104853-3912163e-812f-467f-9c5c-7a0476db5b26.mov



## Test plan
1. Pull the branch
2. Pull the branch from code-intel-extension: https://github.com/sourcegraph/code-intel-extensions/pull/821
3. Run `yarn serve` in code-intel-extensions
4. Run `sg start web-standalone`
5. Sideload the extension with default parameters
6. Check that it works as expected

## App preview:

- [Web](https://sg-web-og-ref-panel-action-button.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-vdcenislxr.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
